packages/techdocs-common/src/stages/publish/openStackSwift.ts

```diff
@@ -93,9 +95,13 @@ export class OpenStackSwiftPublish implements PublisherBase {
    * and the storage client will work.
    */
   getReadiness(): Promise<ReadinessResponse> {
-    return new Promise(resolve => {
-      this.storageClient.getContainer(this.containerName, (err, container) => {
-        if (container) {
+    return new Promise(async resolve => {
```

Comment:
Nit: You could rewrite this to using async-await. There is no real need to doing `new Promise(` here.